src/Support/HigherOrderMessage.php

```diff
@@ -50,6 +52,13 @@ final class HigherOrderMessage
      */
     public $arguments;
 
+    /**
+     * An optional condition that will determine if the message will be executed.
+     *
+     * @var callable|null
```

Comment:
Can you type the format of the callable?

Reply:
Doh! Was meant to come back to this but slipped my mind. Updated 👍